### PR TITLE
Fix tab label

### LINKED ISSUE #1459: Missing label
## What went wrong? 😲
 
I noticed this when reviewing a recent PR

Screenshot 2023-03-30 at 10 05 49 AM

## Expected behaviour 🤔
Correct label should show

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Go to customer requisition, view the stock line edit modal

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #1459 

# 👩🏻‍💻 What does this PR do? 
 
The cause of the problem is that the `useTranslation` hook was not handling default values correctly.
While repairing that, I have refactored the `ModalTabs` usage slightly, as it was relying on the default value in an inelegant way. The implementation was translating twice and this isn't required.

image

### Offroads
- I found that having a logout in the CMD+K useful, and have also added `Admin`
- The `sed` command that was a temporary fix does not work on linux (different sed version) so have changed to perl which works on both platforms.
- Added the `.gitignore` entry for android assets

# 🧪 How has/should this change been tested? 

- Check the stockline edit modal and customer requisition modals, for the tab names.
- Check `yarn generate` for the perl command
- cmd+k for the new actions

## 📃 Documentation

- [ ] Extra cmd+k actions : can update the screenshot in the [tutorial](https://docs.msupply.foundation/docs/introduction/tutorial/)

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "//": "The sed command here is a temporary workaround to fix an issue in graphql-codegen",
-    "generate": "cd ../server && cargo run --bin remote_server_cli -- export-graphql-schema && cd ../client && graphql-codegen --config codegen.yml && find . -type f -name 'operations.generated.ts' -exec sed -i '' s/'graphql-request\\/dist\\/types\\.dom'/'graphql-request\\/src\\/types\\.dom'/g {} +",
+    "generate": "cd ../server && cargo run --bin remote_server_cli -- export-graphql-schema && cd ../client && graphql-codegen --config codegen.yml && find . -type f -name 'operations.generated.ts' -exec perl -i -pe 's/graphql-request\\/dist\\/types\\.dom/graphql-request\\/src\\/types\\.dom/g' {} \\;",
```
Comment (andreievg) [inline client/package.json:22]: I kind of missed this in the previous PR, I would have suggested something like this (with a comment, I think tsconfig.json allows comments):

```diff
diff --git a/client/tsconfig.json b/client/tsconfig.json
index a3acdeb8a..dd1110ab5 100644
--- a/client/tsconfig.json
+++ b/client/tsconfig.json
@@ -9,7 +9,10 @@
       "@common/intl/*": ["packages/common/src/intl/*"],
       "@common/styles": ["packages/common/src/styles"],
       "@common/types": ["packages/common/src/types"],
-      "@common/utils": ["packages/common/src/utils"]
+      "@common/utils": ["packages/common/src/utils"],
+      "graphql-request/dist/types.dom": [
+        "node_modules/graphql-request/src/types.dom"
+      ]
     },
     "resolveJsonModule": true,
     "target": "es2015",
```

**Review by andreievg — APPROVED** [review]
Changes look good, I've noticed there has been whitespace changes in localisation files lately, was wondering where they are from (you probably mentioned it in one of the PR descriptions, sorry If i missed it)

**Comment by mark-prins** [comment]
> I've noticed there has been whitespace changes in localisation files

yeah - the weblate instance has a different opinion about formatting. In the back of my mind I wonder about adding prettier to it, but have not looked yet.

File: client/package.json (hunk 1 above)
Comment (mark-prins) [reply client/package.json:22]: 🤯  that is devious genius! am about to test it..

Comment (mark-prins) [reply client/package.json]: snuck that in to 3467237a0
it's just too good not to

**Pull request merged**